### DISCUSSION TITLE
Add support for OAuth

### DISCUSSION THREAD

**Discussion by xhyrom** [coollabsio/coolify#1755] [Feature Requests]
It would be great to add support for OAuth so you can have one account for everything

**Comment by pksorensen** [comment]
This esists, 
![image](https://github.com/user-attachments/assets/39bf13f8-9a84-46d5-aa5d-3cddc7b85402)


however missing some documentation on how to setup

**Reply by xhyrom** [reply]
this is only for specific providers, not custom one

**Reply by pksorensen** [reply]
ahh, get it now :) also I found the redirect in the source code that i was missing /auth/{provider}/callback  where provider = azure.

**Reply by therealtimex** [reply]
Keycloak is a great provider.
